.changelog/1040.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:bug
+`resource/pingone_gateway`: Correctly include `description` when supplied for `RADIUS` and `LDAP` types.
```

Comment:
I'd probably describe these bug fixes using the error from the user's perspective, like "fixed inconsistent result error when setting description..."